src/hotspot/share/cds/dynamicArchive.cpp

```diff
@@ -349,6 +396,76 @@ class VM_PopulateDynamicDumpSharedSpace: public VM_GC_Sync_Operation {
   }
 };
 
+GrowableArray<ObjArrayKlass*>* DynamicArchive::_array_klasses = nullptr;
+Array<ObjArrayKlass*>* DynamicArchive::_dynamic_archive_array_klasses = nullptr;
```

Comment:
Perhaps a comment to clarify that `_array_klasses` and `_dynamic_archive_array_klasses` only hold the array classes which have element class in static archive would be useful when reading the code later.

Reply:
I've added a comment.